### PR TITLE
build(ci): submodule should sign commits

### PR DESCRIPTION
**Pull request by aali309**
reference: [#916](https://github.com/cryostatio/cryostat-web/issues/916)

### CODE REVIEW

**Review by tthvo — COMMENTED** [review]
Looks good! Just need the key to be generated.

File: .github/workflows/submodule.yml [hunk 1]
```diff
@@ -14,10 +14,17 @@ jobs:
...
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
       with:
         repository: cryostatio/cryostatio.github.io
         token: "${{ secrets.SUBMODULE_TOKEN }}"
+    - name: Import GPG key
+      uses: crazy-max/ghaction-import-gpg@v6
+      with:
+        gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+        passphrase: ${{ secrets.PASSPHRASE }}
```
Comment (andrewazores) [inline .github/workflows/submodule.yml:25]: Rename this to `secrets.GPG_PASSPHRASE` please

**Comment by tthvo** [comment]
@Mergifyio backport gh-pages

**Comment by tthvo** [comment]
Ahh nvmind I don't have permissions^^ But we need to backport this into `gh-pages` as in https://github.com/cryostatio/cryostat-helm/pull/89

**Comment by ebaron** [comment]
Thanks @tthvo! I wonder if this is some kind of default setting with Mergify?

**Comment by tthvo** [comment]
> Thanks @tthvo! I wonder if this is some kind of default setting with **Mergify?**

Yeh exactly! https://docs.mergify.com/commands/restrictions/#command-restriction-format

**Comment by ebaron** [comment]
> > Thanks @tthvo! I wonder if this is some kind of default setting with **Mergify?**
> 
> Yeh exactly! https://docs.mergify.com/commands/restrictions/#command-restriction-format

Perhaps we want something along these lines instead:
```
commands_restrictions:
  backport:
    conditions:
      - sender=@team
```

**Comment by tthvo** [comment]
Right! Thanks! I will make a new issue for this ^^

**Pull request merged**